pkg/apis/openebs/v1alpha1/device_types.go

```diff
@@ -2,21 +2,57 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 )
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 // DeviceSpec defines the desired state of Device
 type DeviceSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
```

Comment:
Can we retain this commit into this file, if the operator-sdk generation needs to be run after modifying this file in future.

Reply:
done.